modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -8,7 +8,18 @@
 = Creating a KubeletConfig CRD to edit kubelet parameters
 
 The kubelet configuration is currently serialized as an Ignition configuration, so it can be directly edited. However, there is also a new
-`kubelet-config-controller` added to the Machine Config Controller (MCC). This allows you to create a `KubeletConfig` custom resource (CR) to edit the kubelet parameters.
+`kubelet-config-controller` added to the Machine Config Controller (MCC). This allows you to use a `KubeletConfig` custom resource (CR) to edit the kubelet parameters.
+
+You should have one `ContainerRuntimeConfig` CR for each machine config pool with all the config changes you want for that pool. You should edit an existing `ContainerRuntimeConfig` CR to modify existing settings or add new settings instead of creating a new CR for each change. It is recommended to create a new CR only to modify a different machine config pool, or for changes that are intended to be temporary, so that you can revert the changes.
+
+You can create multiple `KubeletConfig` CRs, as needed, with a limit of ten. For the first `KubeletConfig` CR, the MCO creates a machine config appended with `kubelet`. With each subsequent CR, the controller creates a new `kubelet` machine config with a numeric suffix. For example, if you have a `kubelet` machine config with a -2 suffix, the next `kubelet` machine config is appended with -3.
```

Comment:
Also you'd better clarify that it is not 10 per pool, it is 10 overall. Customers may ignore this.